Update Russian translations

Numerok opened the pull request:
Added translations for **osc_opacity**, **enable_post_stream_toast** and **checkbox_touchscreen_trackpad** (trackpad called touchpad in Russia, so i had to change it a bit)
Didn't add **checkbox_flip_face_buttons** (don't know exactly what it does) and **audio_config_list** (if I translated it, it would be too different from the original)
But i can translate them, if you need. I'll just translate them literally.

i Also can rearrange xml file to match English file

cgutman commented:
`checkbox_flip_face_buttons` reverses the X/Y and A/B buttons (Nintendo button layout).

How would you translate `audio_config_list`? It doesn't really matter if it's different than the English translation as long as it's an accurate description of the option in Russian.

Can you translate the new `nettest_` strings too?

Numerok commented:
strings.xml:
added flip_face_buttons (there is no "face buttons" definition for gamepads in Russian. Literal translation will be confusing, as it was for me, so i just changed it to be just "x/y and a/b")
added audio_config_list
added nettest_*

arrays.xml:
added audio_config_names

cgutman commented:
Thanks!
